manifests/integreatly-unifiedpush/0.5.2/unifiedpush-operator.v0.5.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,238 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments:
+      - name: unifiedpush-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: unifiedpush-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: unifiedpush-operator
+            spec:
+              containers:
+              - command:
+                - unifiedpush-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: unifiedpush-operator
+                image: quay.io/integreatly/aerogear-unifiedpush-operator:v0.5.2
```

Comment:
Is this image built? I think that could be the cause of the e2e test failure?

Reply:
Yes, that is more than likely :)

I just merged the PR for aerogear/unifiedpush-operator repo, and image is being built right now.
I'll trigger e2e tests again once the image is in quay.